### DISCUSSION TITLE
Is there any method to make my contributions and activity private  AND my Pinned repositories public AND the Follow function enabled on the overview page??

### DISCUSSION THREAD

**Discussion by ghost** [community/community#34557] [Other Feature Feedback, Questions, & Ideas]
When I set "Make profile private and hide activity" enabled,  why both my Pinned repositories AND the Follow function are disabled on the overview page?
And when I set "Make profile private and hide activity" disabled, why others can see my contributions and activity in 2019 but can not see my contributions and activity in 2021?

**Comment by jfullstackdev** [comment]
it's hard to tell those commits that can't be seen when you make your profile public again

> Pinned repositories AND the Follow

it's private that's why, not like Twitter but more of like Messenger that when you disabled your active status, to be fair, you should not be seeing other's active status too

well, it's all about what GitHub choice,
